### ISSUE TITLE
How to connect on local network and also delete entries?

### ISSUE THREAD

**Issue by exsammy** [opensource-emr/hospital-management-emr#68]
V3 installed and working properly.

How do I connect other computers on the same local network to work in the office?

How to delete entries? Once created there is no delete button, only edit

**Comment by opensource-emr** [comment]
You can watch this video how to configure inside IIS https://youtu.be/HmAAbFiPOKw

Danphe follows strict data rules , so delete and edit are not available everywhere. So for example if you enter wrong accountin entry , you can not delete you will need to make contra entry.
 This is just to make sure we whatever  data was entered its tracked.

If you want to delete data you will need to do from back end.

**Comment by opensource-emr** [comment]
Closing...This issue

**Comment by CreativeOthman** [comment]
How to start with fresh DB. Too many informations ,Thanks

**Comment by CreativeOthman** [comment]
yes its posible to delete the entries from backend,i think two steps are involve , 

1. run the cleanUpScript on your main EMRDB 
2. delete the entries in the lab,radiology,clinical,phamacy tables,(you need to be patient)

**Comment by opensource-emr** [comment]
If you wish to clean the DB from scratch you can check out this clean up script https://github.com/opensource-emr/cleanupDanpheDM .

Please be careful before running this script as the deletes all transaction data. Would suggest to keep a back of your DB before executing this script.
